index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="Header">
+   <header>
+    <nav class="nav-bar">
+        <a href="#">Brenda Lutiku</a>
+        <img src="./images/Enabled.png" alt="" class="menu">
+    </nav>
+   </header>
+   <section class="container">
+      <h1 class="headin">I’m Brenda.Glad to see you!</h1>
+      <p class="paragrap">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+    <img src="./images/Supporting Text.png" alt="" class="text-supprt">
+      <div class="second-container">
+        <img src="./images/social media icons.png">
+       </div> 
```

Comment:
- Kindly I suggest you not use pictures because these we link to your profiles 

  - The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).

Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.


  - The button (if the template has one) is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design).

Apart from being semantically correct, the a tag will let you connect to other sections of the website, using href.

- Add also some space between them :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/228687965-e7cbbee5-e81c-4a9e-bf7b-4fc8724e5b3e.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/228688005-1003f6fa-783c-4d27-ad9e-182ee8a648a1.png)